### PR TITLE
Add optional longitudinal residual to kinkal fits

### PR DESCRIPTION
**Pull request by bonventre**
Enabled in CentralHelix and KinematicLine fits

### CODE REVIEW

File: CosmicReco/fcl/prolog.fcl [hunk 1]
```diff
@@ -78,7 +78,7 @@ CosmicReco : { @table::CosmicReco
 
    CosmicTrackFinder : { #split into Loose and tight below for efficieny studies
       module_type                   : CosmicTrackFinder
-      ComboHitCollection     : "makePH"
+      ComboHitCollection     : "makeSH"
```
Comment (brownd1978) [inline CosmicReco/fcl/prolog.fcl:81]: If cosmicreco never uses panel hits I suggest overwriting makePH with RecoNullProducer as is done [here](https://github.com/Mu2e/Offline/blob/dd31c82e4e5c4edeea674b8a894c24acffccd39f/TrkHitReco/fcl/prolog.fcl#L127), then no time will be wasted on calling it.

**Review by brownd1978 — CHANGES_REQUESTED** [review]
This looks good, there are just a few small things I'd appreciate you looking at.

File: CosmicReco/fcl/prolog.fcl (hunk 1 above)
Comment (bonventre) [reply CosmicReco/fcl/prolog.fcl:81]: Added dummy producers to prolog, will have to change in Production to implement in reco

Comment (brownd1978) [reply CosmicReco/fcl/prolog.fcl:81]: You could name these instances makePH, makeSTH, then no production change is neeeded

File: CosmicReco/fcl/prolog.fcl [hunk 2]
```diff
@@ -167,7 +174,7 @@ CosmicReco : { @table::CosmicReco
       module_type : CosmicTrackDiag
       StrawHitCollection : "makeSH"
       ComboHitCollection : "makeSH"
-      PanelHitCollection : "makePH"
+      PanelHitCollection : "makeSH"
```
Comment (brownd1978) [inline CosmicReco/fcl/prolog.fcl:177]: Does PanelHitCollection still make sense for this module?

**Review by brownd1978 — APPROVED** [review]
I suggest a few possible improvements.

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Pull request merged**